src/Entity/AdminUser.php

```diff
@@ -4,21 +4,17 @@
 
 namespace Sylius\RbacPlugin\Entity;
 
+use Doctrine\ORM\Mapping\MappedSuperclass;
+use Doctrine\ORM\Mapping\Table;
 use Sylius\Component\Core\Model\AdminUser as BaseAdminUser;
+use Sylius\Component\Core\Model\AdminUserInterface;
 
-/** @final */
-class AdminUser extends BaseAdminUser implements AdminUserInterface
+/**
+ * @MappedSuperclass()
+ * @Table(name="sylius_admin_user")
+ * @final
+ */
+class AdminUser extends BaseAdminUser implements AdministrationRoleAwareInterface, AdminUserInterface
```

Comment:
If you add `AdminUser` in `tests/Application/src/Entity`, do we need this class?